WordPress/wordpress-playground
# [Data Liberation] Move Markdown importer to a separate package

Moves the Markdown importer to a `data-liberation-markdown` package so
that it can be shipped as a separate `.phar` file and downloaded only
when needed.

 ## Testing instructions

This only moves code around. To test, confirm the CI PHP unit tests keep
working.

A part of:

* https://github.com/WordPress/wordpress-playground/pull/2080
* #1894
